tests/acceptance/features/coreApiSharePublicLink1/changingPublicLinkShare.feature

```diff
@@ -191,7 +191,7 @@ Feature: changing a public link share
       | 1               | 100             |
       | 2               | 200             |
 
-
+  @skipOnReva # reva doesn't have a pre-created admin user
```

Comment:
there should be `admin` user, right?

Reply:
I didn't look in too deep but we have similar test skipped because of the same reason 

https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareManagementBasicToShares/createShareToSharesFolder.feature#L239

Maybe there's no "admin" user like we have in the ocis?

Reply:
Currently ` And admin has created folder "/PARENT"` this step fails in reva with `401`

Reply:
Are these newly added test scenarios?

Reply:
yes, the scenarios skipped in this PR are newly added ones. 
this however isn't https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareManagementBasicToShares/createShareToSharesFolder.feature#L239

Reply:
Okey, then I won't trouble you more :sweat_smile: